pyanaconda/installation.py

```diff
@@ -100,11 +101,15 @@ def _prepare_configuration(storage, payload, ksdata):
     services_dbus_tasks = services_proxy.InstallWithTasks()
     os_config.append_dbus_tasks(SERVICES, services_dbus_tasks)
 
-    os_config.append(Task("Configure keyboard", ksdata.keyboard.execute))
+    # schedule keyboard configuration
+    localization_proxy = LOCALIZATION.get_proxy()
+    os_config.append(Task("Configure keyboard",
+                          keyboard.write_keyboard_config,
```

Comment:
The function `write_keyboard_config` should be transformed into a DBus task. This could be just a temporary solution.